### PR TITLE
fix(core): create new general system error

### LINKED ISSUE #140: Fix Syntax-Error message
The output of `function system_error()` should be changed to the BS3-Style - also the <head> section should be updated.

### PR DESCRIPTION
**Pull request by derchrisuk**
closes #140
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/webSPELL/webSPELL/pull/231%23issuecomment-94859081%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/231%23issuecomment-94874540%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/231%23issuecomment-94874794%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/231%23issuecomment-94875207%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/231%23issuecomment-94913190%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/231%23issuecomment-94913405%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/231%23discussion_r28805667%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/231%23discussion_r28807422%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/231%23discussion_r28813317%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/231%23discussion_r28813516%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/231%23discussion_r28813518%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL/pull/231%23issuecomment-94859081%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22What%20about%20replacing%20the%20default%20php%20error%20handler%20with%20this%22%2C%20%22created_at%22%3A%20%222015-04-21T16%3A19%3A51Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%2C%20%7B%22body%22%3A%20%22So%20instead%20of%20just%20using%20exceptions%20for%20the%20few%20system_error%2C%20use%20it%20globally%3F%22%2C%20%22created_at%22%3A%20%222015-04-21T17%3A08%3A28Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%2C%20%7B%22body%22%3A%20%22yes%20but%20it%20was%20just%20an%20idea%20but%20should%20be%20post%204.3%5Cr%5Cn%22%2C%20%22created_at%22%3A%20%222015-04-21T17%3A09%3A14Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%2C%20%7B%22body%22%3A%20%22Think%20so%20too.%20For%20now%20we%20should%20be%20good%20with%20the%20current%20modifications.%5Cr%5CnAt%20least%20they%20are%20already%20in%20the%20same%20format.%5Cr%5CnSo%20for%20post-4.3%20we%20can%20add%20to%20that%20and%20make%20a%20full%20exception%20error%20handling%20out%20of%20it.%22%2C%20%22created_at%22%3A%20%222015-04-21T17%3A10%3A24Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222015-04-21T19%3A30%3A45Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-04-21T19%3A32%3A03Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20b51bc8222934493dc216fe2e8b8360cc750ac847%20_error.php%2074%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL/pull/231%23discussion_r28807422%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22The%20keywords%20meta%20tag%20is%20deprecated%20by%20most%20search%20engines%2C%20should%20be%20removed.%20ref%20%23227%20%22%2C%20%22created_at%22%3A%20%222015-04-21T18%3A25%3A59Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%2C%20%7B%22body%22%3A%20%22This%20has%20been%20taken%20from%20the%20current%20index%20page%2C%20so%20will%20leave%20it%20for%20now%22%2C%20%22created_at%22%3A%20%222015-04-21T19%3A28%3A39Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222015-04-21T19%3A30%3A45Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20_error.php%3AL1-127%22%7D%2C%20%22Pull%20b51bc8222934493dc216fe2e8b8360cc750ac847%20_error.php%2084%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL/pull/231%23discussion_r28805667%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Can%20be%20done%20using%20%60text-center%60%5Cr%5Cnhttp%3A//getbootstrap.com/css/%23type-alignment%22%2C%20%22created_at%22%3A%20%222015-04-21T18%3A08%3A38Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-complete%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222015-04-21T19%3A30%3A46Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20_error.php%3AL1-127%22%7D%7D%2C%20%22approved%22%3A%20%7B%22https%3A//github.com/derchrisuk%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/derchrisuk'></a>

- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL/pull/231#issuecomment-94859081'>General Comment</a></b>
- <a href='https://github.com/Phhere'></a> What about replacing the default php error handler with this
- <a href='https://github.com/derchrisuk'></a> So instead of just using exceptions for the few system_error, use it globally?
- <a href='https://github.com/Phhere'></a> yes but it was just an idea but should be post 4.3
- <a href='https://github.com/derchrisuk'></a> Think so too. For now we should be good with the current modifications.
At least they are already in the same format.
So for post-4.3 we can add to that and make a full exception error handling out of it.
- [x] <a href='#crh-comment-Pull b51bc8222934493dc216fe2e8b8360cc750ac847 _error.php 84'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL/pull/231#discussion_r28805667'>File: _error.php:L1-127</a></b>
- <a href='https://github.com/Pascalmh'></a> Can be done using `text-center`
http://getbootstrap.com/css/#type-alignment
- [x] <a href='#crh-comment-Pull b51bc8222934493dc216fe2e8b8360cc750ac847 _error.php 74'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL/pull/231#discussion_r28807422'>File: _error.php:L1-127</a></b>
- <a href='https://github.com/Pascalmh'></a> The keywords meta tag is deprecated by most search engines, should be removed. ref #227
- <a href='https://github.com/derchrisuk'></a> This has been taken from the current index page, so will leave it for now

<a href='https://www.codereviewhub.com/webSPELL/webSPELL/pull/231?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/webSPELL/webSPELL/pull/231?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/webSPELL/webSPELL/pull/231?approve=0'></a>&nbsp;<a href='https://github.com/webSPELL/webSPELL/pull/231'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by webSPELL-org** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/webSPELL/webSPELL/pull/231'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by Phhere** [comment]
What about replacing the default php error handler with this

**Comment by derchrisuk** [comment]
So instead of just using exceptions for the few system_error, use it globally?

**Comment by Phhere** [comment]
yes but it was just an idea but should be post 4.3

**Comment by derchrisuk** [comment]
Think so too. For now we should be good with the current modifications.
At least they are already in the same format.
So for post-4.3 we can add to that and make a full exception error handling out of it.

File: _error.php [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+<meta name="generator" content="webSPELL">
+
+<!-- Head & Title include -->
+<title>webSPELL - Error</title>
+<link href="components/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
+<link href="_stylesheet.css" rel="stylesheet">
+<style>
+/* centered columns styles */
+.row-centered {
+    text-align:center;
```
Comment (Pascalmh) [inline _error.php]: Can be done using `text-center`
http://getbootstrap.com/css/#type-alignment

File: _error.php [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        $info = 'webSPELL';
+    }
+    die('<!DOCTYPE html>
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<meta charset="utf-8">
+<meta name="description" content="Clanpage using webSPELL 4 CMS">
+<meta name="author" content="webspell.org">
+<meta name="keywords" content="webspell, webspell4, clan, cms">
```
Comment (Pascalmh) [inline _error.php]: The keywords meta tag is deprecated by most search engines, should be removed.

Comment (derchrisuk) [inline _error.php]: This has been taken from the current index page, so will leave it for now

Comment (derchrisuk) [inline _error.php]: :+1:<a href='#crh-update-none'></a>

**Comment by derchrisuk** [comment]
:+1:<a href='#crh-update-none'></a>

File: _error.php (hunk 1 above)
Comment (derchrisuk) [inline _error.php]: :+1:<a href='#crh-update-complete'></a>

**Comment by derchrisuk** [comment]
:shipit:

**Pull request merged**